README.md

```diff
@@ -8,6 +8,25 @@ Gogs, Gitea).
 [ActivityPub]: https://www.w3.org/TR/activitypub/
 [ActivityStream2]: https://www.w3.org/TR/activitystreams-core/
 
+## Updates
+### ForgeFed Community Forum
+
+In order to consolidate development and community discussions, there is now a **public web forum** on the socialhub.network website. That forum is the home to ActvityPub and several projects that use ActvityPub as their federation protocol; and so it is a natural fit for ForgeFed. Everyone is invited to use that forum instead of this GitHub repo or the mailing list.
+
+Thanks to SocialHub and FeNeAs for making that possible.
+
+Web Forum: https://socialhub.network/c/forgefed
+### ForgeFed in the Fediverse
```

Comment:
I'd suggest adding a blank line above this heading and the next one.

Reply:
fixed in https://github.com/forgefed/forgefed/pull/26/commits/fe575c02bbec99acf8ba55bf3cbe405b504bc0f2 thanks for the feedback! :3